src/test.ml

```diff
@@ -358,6 +359,72 @@ let test_agg_nullable = [
   |} [ attr' "result" Int; ][];
 ]
 
+let test_ambiguous = [
+  tt "CREATE TABLE test21 (id INT, column_a TEXT, column_b BOOL)" [] [];
+  tt "CREATE TABLE test22 (id INT, column_d INT)" [] [];
+  wrong "select id from test21 join test22 on test21.id = test22.id order by id";
+  tt "select test21.id from test21 join test22 on test21.id = test22.id order by id" [
+    attr' ~nullability:(Nullable) "id" Int;
+  ] [];
+  wrong "select test21.id from test21 join test22 on test21.id = test22.id where id > 2 order by id";
```

Comment:
please put comment from the PR here why this one doesn't fail

Reply:
Okay, but it fails. I cal here the "wrong" function. I means assert fail.

Reply:
i meant comment this difference in behaviour between where vs order by

Reply:
Added comment